### DISCUSSION TITLE
Multi Factor Auth after defining Super User

### DISCUSSION THREAD

**Discussion by sudo-joe** [nzymedefense/nzyme#1142] [Questions] [answered]
Hello,

after initial setup Nzyme asks to setup a the Super User, and after that a QR code is shown (for YubiKey and Google Authenticator App, but I am not going to use them).

Now many lines of text in about 5-6 raws are shown and on the next screen, 6 boxes....

Question:
Should it "work", if one of the (entire) words of the list is entered in this box?

Thanks in advance for any help!

PS:
I am on:  [2.0.0-alpha.14](https://github.com/nzymedefense/nzyme/releases/tag/2.0.0-alpha.14)

**Comment by lennartkoopmann** [comment, marked as answer]
Hi,

you have to use a TOTP app like Authy, Yubico or Microsoft or Google Authenticator. The rows of text you see are your backup codes and the screen should describe what they are for.

You enter your TOTP number into the fields for the numbers or follow the "Use Recovery Code" link to use a recovery code.

**Comment by sudo-joe** [comment]
Thank's Lennart for the fast help!

It was the first time for me to work with a TOTP app. Everything does work perfectly and I am using FreeOTP
